flybywire-aircraft-a320-neo/nl-NL.locPak

```diff
@@ -0,0 +1,25 @@
+{
+  "LocalisationPackage": {
+    "Language": "nl-NL",
+    "Strings": {
+      "AIRCRAFT.DESCRIPTION": "Airbus lanceerde hun eerste A320 in maart, 1984. Bijna 26 jaar later, in 2010, kondigden ze de A320neo (\"new engine option\") aan als het nieuwste lid van de familie, om hun lijn van wereld leidende, narrow-body, twin-turbofan vliegtuigen te vernieuwen en in stand te houden.<\/br><\/br>Na een uitgebreid ontwikkelingsproces is de A320neo officieel in gebruik genomen bij Lufthansa, begin 2016.  De A320neo werd al gauw het snelst verkopende commerciële vliegtuig ooit, met meer dan 7.400 bestelde vliegtuigen door meer dan 115 vliegmaatschappijen, tot op heden.<\/br><\/br>De A320neo zet de door de originele A320 (welke het eerste vliegtuig was, naast de Concorde, welke gebruik maakte van “Fly-By-Wire” besturing voor regulier gebruik) vastgestelde traditie voort door 95 procent van de vliegtuigromp te delen met de voorgaande modellen. Het was de introductie van twee nieuwe generatie motor types, samen met de gesloten “Sharklet” vleugeltips, wat de neo echt onderscheidde.<\/br><\/br>Brandstofbesparing, verlengd bereik, verhoogde maximale vracht belasting en een verkorte landingsafstand; deze belangrijke ontwikkelingen, samen met een enorme terugname in geluid en CO₂ emissie, bewijzen waarom de A320neo niet alleen een commercieel succes is, maar ook een aanzienlijke stap vooruit is op de weg van de luchtvaart naar een grotere milieuverantwoordelijkheid.",
+      "AIRCRAFT.UI_MANUFACTURER": "FlyByWire Simulations",
+      "AIRCRAFT.UI_MODEL": "A320neo (LEAP)",
+      "AIRCRAFT.UI_VARIATION": "Airbus",
+      "COCKPIT.TOOLTIPS.LIGHTING_KNOB_R_PFD_DECREASE": "Helderheid PFD rechts verlagen",
+      "COCKPIT.TOOLTIPS.LIGHTING_KNOB_R_PFD_INCREASE": "Helderheid PFD rechts verhogen",
```

Comment:
```suggestion
      "COCKPIT.TOOLTIPS.LIGHTING_KNOB_R_PFD_DECREASE": "Helderheid rechter PFD verlagen",
      "COCKPIT.TOOLTIPS.LIGHTING_KNOB_R_PFD_INCREASE": "Helderheid rechter PFD verhogen",
```

Reply:
that would change the logic of the tooltip here, in relation to the other PDF

Reply:
ah, it's changed, i can't do anything about it, right?

Reply:
Interesting that the other side is not in this file. They would be:
`COCKPIT.TOOLTIPS.LIGHTING_KNOB_L_PFD_DECREASE`: Helderheid linker PFD verlagen
`COCKPIT.TOOLTIPS.LIGHTING_KNOB_L_PFD_INCREASE`: Helderheid linker PFD verhogen

Reply:
the reason we're putting in these two key/value pairs is that in the original model, there's a typo/mistake. If the tooltip is not provided in the locPak the knob reads something like:

Increase brightness right PFD
Decrease brightness left PFD

whereas the know should increase/decrease only the right PFD

Reply:
I see. So you're saying that the original translation is of the form "Helderheid PFD links verlagen", and my suggestion changes the form only for the right one. Irregardless of my translation being better or not, I would still suggest putting the left PFD into the translation file as well. Otherwise there's bound to be some point in the future where someone does change the right one, but not the left one, as it is nowhere to be found.

I will however approve the PR in any case. Just a suggestion! 😀

Reply:
Exactly. I could add the left one as well if you feel that it makes sense! Could probably get it done by tomorrow. :)

Reply:
Does this PR fix https://github.com/flybywiresim/a32nx/issues/5575 ?

Reply:
i don’t think so, i’m happy to look into it though. didn’t encounter that problem yet.

Reply:
also, @davidwalschots , it might take a little longer with adding the left PFD tooltips - still up for doing it, but i (again) underestimated how much communication is needed for all that :)

Reply:
added the tooltips for the left side - gotta be honest, the coder in me doesn't really like the redundancy, but I trust your feeling on this :)